packages/ag-charts-community/src/chart/series/dataModelSeries.ts

```diff
@@ -24,6 +24,7 @@ export abstract class DataModelSeries<
     protected dataModel?: DataModel<any, any, any>;
     protected processedData?: ProcessedData<any>;
     protected showFocusBox: boolean = true;
+    protected clipFocusBox: boolean = true;
```

Comment:
This is fine for this release.

However, ideally this should be part of a utility function like `computeBarFocusBounds` and `computeMarkerFocusBounds` to avoid needing to override this value for these marker-based and bar-based chart type. That would require a significant refactor however, so please file a ticket in the backlog to tidy this up post-release. (In the fact, the showFocusBox property is obsolete too, because it's always `true`).